source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst

```diff
@@ -79,10 +79,10 @@ rescan is similar, but not identical to the behaviour of initial seedlist
 discovery.  Periodic scan MUST follow these rules:
 
 - The driver will query the DNS server for SRV records on
-  ``{hostname}.{domainname}``, prefixed with the SRV service name
-  and protocol. The SRV service name is provided in the ``srvServiceName`` URI option and
-  defaults to ``mongodb``. The protocol is always ``tcp``. After prefixing, the URI
-  should look like: ``_{srvServiceName}._tcp.{hostname}.{domainname}``.
+``{hostname}.{domainname}``, prefixed with the SRV service name and protocol.
+  The SRV service name is provided in the srvServiceName_ URI option and
+  defaults to ``mongodb``. The protocol is always ``tcp``. After prefixing, the
+  URI should look like: ``_{srvServiceName}._tcp.{hostname}.{domainname}``.
```

Comment:
Should we add the validation for `domainname` consisting of `domain` + TLD here as well?

Reply:
This seems outside the scope of this PR. If you'll be creating a DRIVERS ticket for https://github.com/mongodb/specifications/pull/1069/files#r724565721 would you like to lump this in as well?